opts/opts.go

```diff
@@ -188,3 +188,35 @@ func ValidateLabel(val string) (string, error) {
 	}
 	return val, nil
 }
+
+// ValidateSysctl validates an sysctl and returns it.
+func ValidateSysctl(val string) (string, error) {
+	validSysctlMap := map[string]bool{
+		"kernel.msgmax":          true,
```

Comment:
@mysterlune Do you know if these sysctls are namespaced?  If  you set them inside of the container, do the settings  show up outside of the container?  If not, then we would add them, if yes then we will not add them.

You could test this by running a --privileged container.